app/views/application/choices/provider.njk

```diff
@@ -6,7 +6,7 @@
 
 {% block primary %}
 
-<p class="govuk-body">You can preview a list of training providers and courses currently available on Apply for teacher training <a href="/apply/providers">here</a>.</p>
+<p class="govuk-body">You can <a href="/apply/providers">preview a list</a> of training providers and courses currently available on Apply for teacher training.</p>
```

Comment:
:+1:
Can we make the full link: `preview a list of training providers` so it makes sense in isolation.